docs/devctr-image.md

```diff
@@ -80,7 +80,10 @@ registry. The Firecracker CI suite must also be updated to use the new image.
     ```
 
 1. Tag the new image with the next available version and the architecture
-   you're on.
+   you're on. Note that this will not always be "current version in devtool + 1",
+   as sometimes that version might already be used on feature branches. Always
+   check the "Image Tags" on [the fcuvm repository](https://gallery.ecr.aws/firecracker/fcuvm)
+   to make sure you do not accidentally overwrite existing image.
 
```

Comment:
Maybe can add this as a sanity check before push i.e. this should fail: 

```sh
docker pull public.ecr.aws/firecracker/fcuvm:v47

Error response from daemon: manifest for public.ecr.aws/firecracker/fcuvm:v47 not found: manifest unknown: Requested image not found
```

Reply:
Good point, added it!